platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/Light.java

```diff
@@ -0,0 +1,128 @@
+// This file is generated. Edit android/platform/scripts/generate-style-code.js, then run `make android-style-code`.
+
+package com.mapbox.mapboxsdk.style;
+
+import android.support.annotation.UiThread;
+import com.mapbox.mapboxsdk.style.layers.TransitionOptions;
+import com.mapbox.mapboxsdk.style.layers.PropertyValue;
+
+/**
+ * 
+ *
+ * @see <a href="https://www.mapbox.com/mapbox-gl-style-spec/#light>">The online documentation</a>
+ */
+@UiThread
+public class Light {
+
+  private long nativePtr;
+
+  /**
+   * Creates a Light.
+   *
+   * @param nativePtr pointer used by core
+   */
+  public Light(long nativePtr) {
+    this.nativePtr = nativePtr;
+  }
+
+  /**
+   * Set the Anchor property.
+   *
+   * @param String value of anchor
+   */
+  public void setAnchor(String anchor) {
+    nativeSetAnchor(new PropertyValue<>("anchor", anchor));
+  }
+
+  /**
+   * Set the Position property.
+   *
+   * @param Float[] value of position
+   */
+  public void setPosition(Float[] position) {
```

Comment:
Don't like the Float[] signature here, would opt in using PointF instead cfr other places in SDK.